ISHPermissionKit/Requests/ISHPermissionRequestEventStore.m

```diff
@@ -55,7 +55,8 @@ - (void)requestUserPermissionWithCompletionBlock:(ISHPermissionRequestCompletion
     
     [self.eventStore requestAccessToEntityType:[self entityType] completion:^(BOOL granted, NSError *error) {
         dispatch_async(dispatch_get_main_queue(), ^{
-            completion(self, granted ? ISHPermissionStateAuthorized : ISHPermissionStateDenied, error);
+            NSError *externalError = (error.code == EKErrorEventStoreNotAuthorized) ? nil : error;
```

Comment:
I'd rather make this `(error && (error.code == EKErrorEventStoreNotAuthorized))`. This isn't a real issue assuming `EKErrorEventStoreNotAuthorized` is not `0`, but this could change theoretically.